### ISSUE TITLE
Affix put in red but it's the good one

### ISSUE THREAD

**Issue by Saharash** [josdemmers/Diablo4Companion#98]
Hi, i have a problem as you can see on screen, the 25% resistance to all elements is red, but the build it's the great one. i have imported from maxbuild

![image](https://github.com/josdemmers/Diablo4Companion/assets/34897211/5eef4f52-b95b-40de-9745-fb3375a5ab8b)

**Comment by Saharash** [comment]
Oh ok sir didn’t knew that. Thanks for the answer and the job on this project it’s a really good job. 

Do you think you could add the possibility to check affix on the compare list ? Like actually it show either on equipped item either on the compare item page, maybe that will require to check 2 images at once I don’t know if possible.

And

When I’m talking to occultist or something the affix isn’t detected I have to leave the page to ser it working

**Comment by josdemmers** [comment]
> Oh ok sir didn’t knew that. Thanks for the answer and the job on this project it’s a really good job.

Thank you, glad you like it!

> Do you think you could add the possibility to check affix on the compare list ? Like actually it show either on equipped item either on the compare item page, maybe that will require to check 2 images at once I don’t know if possible.

It's not possible at this time to handle two items at the same time. No plans to add this currently.

> When I’m talking to occultist or something the affix isn’t detected I have to leave the page to see it working

The item is detected by looking for the shift key. You don't have that at the occultist.

With shift-key in inventory
![image](https://github.com/josdemmers/Diablo4Companion/assets/8627157/b72ee5ae-7fe3-4a9b-908f-888941321a1d)

Without shift-key on character
![image](https://github.com/josdemmers/Diablo4Companion/assets/8627157/2b3e35cd-c4f7-4e72-a4e7-617ecc0bf1b6)

You could try adding a space image and save it here:
`.\Diablo4Companion\Images\1440p_SMF_en\Tooltips\tooltip_kb_space.png`
![image](https://github.com/josdemmers/Diablo4Companion/assets/8627157/4f942381-a396-4613-b439-bf70a9ddf5b0)

For the `inherent affixes`, I'll probably work on that next weekend.

**Comment by Saharash** [comment]
Thanks, i have an issue actually, armor affix is red but i have it been in the app 
![image](https://github.com/josdemmers/Diablo4Companion/assets/34897211/15948b0d-fc7c-4165-8c05-2a9e8b23defb)

It's doing same for all my chest with armor

**Comment by josdemmers** [comment]
That's a bug some affixes have problems with. It's caused by the `Advanced tooltip information` setting ingame.
As a temporarily fix you could disable that, or look at the debug tab and see what affix is actual detected and temporarily add that one.

For armor it's probably ice armor
![image](https://github.com/josdemmers/Diablo4Companion/assets/8627157/2f64ae76-a041-4bb2-a787-92a6069c494e)

**Comment by Saharash** [comment]
> You could try adding a space image and save it here: `.\Diablo4Companion\Images\1440p_SMF_en\Tooltips\tooltip_kb_space.png` ![image](https://private-user-images.githubusercontent.com/8627157/332285683-4f942381-a396-4613-b439-bf70a9ddf5b0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0NjQ2NDMsIm5iZiI6MTcxNjQ2NDM0MywicGF0aCI6Ii84NjI3MTU3LzMzMjI4NTY4My00Zjk0MjM4MS1hMzk2LTQ2MTMtYjQzOS1iZjcwYTlkZGY1YjAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjNUMTEzOTAzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTYwY2RmODNjZmI4MzE2ZjA5NjBmMjNjMjgwYjU5OWE5OWEwZWU5M2ExMGRlNTNhZWUxMDYzYTMyYzhiMjQxNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.2M1KGvTrfq2-mlCcEnjCucj3GxxHbrefdg6Atl-imNc)
> 

I just did that btw and it's working now like a charm ! Thanks sir.

> For armor it's probably ice armor ![image](https://private-user-images.githubusercontent.com/8627157/333040763-2f64ae76-a041-4bb2-a787-92a6069c494e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY0NjQ2NDMsIm5iZiI6MTcxNjQ2NDM0MywicGF0aCI6Ii84NjI3MTU3LzMzMzA0MDc2My0yZjY0YWU3Ni1hMDQxLTRiYjItYTc4Ny05MmE2MDY5YzQ5NGUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDUyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA1MjNUMTEzOTAzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzdmYTI0ZWVkNWViYTRjYTBlMzk0N2IwZTA5NTAzZGRlN2IxOTY1ZjFiM2EyOTE3MzM5NDE4NzUzYmIxOTZmMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.rwIlOKAXOImiTjFprtOxVR4uNmponhSWwikzS6GEYcQ)
> 
> Will look into it later how I can get this working while `advanced tooltip information` is enabled.


OFC i guess it's icearmor problem that what i have: 
![image](https://github.com/josdemmers/Diablo4Companion/assets/34897211/eb0cde5e-143c-4fc6-8a6c-21023404324b)

Thanks for your work sir!

**Comment by josdemmers** [comment]
> intelligence is green because i need it so it show good almost, but the best would be Cooldown Reduction in Greater Affix in my build, so would be dope if we can have for example yellow color to say that it's good but not at the good spot

Have some bugs and other features I'm planning to focus on first but will write it down.
Probably implement it by keeping the same colors but adding something to the circle instead to indicate which affixes you prefer as a greater affix.

**Comment by Saharash** [comment]
> > intelligence is green because i need it so it show good almost, but the best would be Cooldown Reduction in Greater Affix in my build, so would be dope if we can have for example yellow color to say that it's good but not at the good spot
> 
> Have some bugs and other features I'm planning to focus on first but will write it down. Probably implement it by keeping the same colors but adding something to the circle instead to indicate which affixes you prefer as a greater affix.

Yeah it could be insane in that way, take your time, thanks for the work

**Comment by Saharash** [comment]
> Inherent affixes from maxroll builds are now support in v3.4.6.0.

Sorry to answer this, but does it work too for D4 build import?

thanks for update!

**Comment by josdemmers** [comment]
Yes works for d4builds as well. No extra changes were needed for d4builds.
